### ISSUE TITLE
[Bug] Integration won't load when SMART is enabled

### ISSUE THREAD

**Issue by conorlap** [tomaae/homeassistant-openmediavault#110]
## Describe the issue

Integration won't load when SMART is enabled. It loads fine when I disable SMART.

## Software versions

 - Home Assistant version: 2023.4.6
 - openmediavault integration version: v1.4.0
 - openmediavault software version: 6.3.10-2 

## Traceback/Error logs
```

Logger: homeassistant.config_entries
Source: custom_components/openmediavault/apiparser.py:47
Integration: OpenMediaVault ([documentation](https://github.com/tomaae/homeassistant-openmediavault), [issues](https://github.com/tomaae/homeassistant-openmediavault/issues))
First occurred: 0:51:37 pm (1 occurrences)
Last logged: 0:51:37 pm

Error setting up entry JayNAS for openmediavault
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 383, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/openmediavault/__init__.py", line 35, in async_setup_entry
    await controller.async_update()
  File "/config/custom_components/openmediavault/omv_controller.py", line 184, in async_update
    await self.hass.async_add_executor_job(self.get_smart)
  File "/usr/local/lib/python3.10/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/openmediavault/omv_controller.py", line 350, in get_smart
    tmp_data = parse_api(
  File "/config/custom_components/openmediavault/apiparser.py", line 145, in parse_api
    data = fill_vals(data, entry, uid, vals)
  File "/config/custom_components/openmediavault/apiparser.py", line 277, in fill_vals
    data[uid][_name] = from_entry(entry, _source, default=_default)
  File "/config/custom_components/openmediavault/apiparser.py", line 47, in from_entry
    ret = int(ret)
ValueError: invalid literal for int() with base 10: '0/9'

```

**Comment by tomaae** [comment]
"0/9"? looks like another non-standard smart response.
Can you give me debug?

**Comment by tomaae** [comment]
there is no such error in debugs

**Comment by conorlap** [comment]
[debug_log.txt](https://github.com/tomaae/homeassistant-openmediavault/files/11344067/debug_log.txt)

Apologies- think I got the error this time in log!

**Comment by tomaae** [comment]
Its issue in OMV API, it returns empty string in integer field. Added workaround for it and re-released it under 1.4.0

**Comment by conorlap** [comment]
Excellent, the latest version is now working great. Thanks so much for this great integration!! 🥳
